docs/dashboard/bugtracker/index.md

```diff
@@ -34,7 +34,7 @@ App Center has bug tracker integration for the crashes service. Users can be qui
       1. Login with your Azure DevOps credentials and click **Accept** when prompted on app authorization.
       2. Select which Azure DevOps projects to integrate the bug tracker with and click **Next**.
       3. Toggle **On** for **Auto create ticket**, and fill in the fields for **Area** and **Default Payload**, and click **Add**:
-         - Default payload is an optional field to fill in for use in work items. For example, `{"System.IterationPath": "Area\\Iteration 1", "System.AssignedTo": "Fabrikam"}`. Refer to the [work item types API](/rest/api/vsts/wit/work%20item%20types) for additional information.
+         - Default payload is an optional field to fill in for use in work items. For example, `{"System.IterationPath": "Area\\Iteration 1", "System.AssignedTo": "Fabrikam"}`. Refer to the [work item types API](/rest/api/vsts/wit/) for additional information.
```

Comment:
The url should be `/rest/api/azure/devops/wit/work-item-types`

Reply:
Changed, please review.